app/login/page.tsx

```diff
@@ -0,0 +1,15 @@
+"use client";
```

Comment:
Use SSR unless client rendering is absolutely needed

Reply:
To use "useAuthState" it has to be unfortunately.